scala-parallel-collections/src/test/scala/zio/prelude/scalaparallelcollections/HashJvmSpec.scala

```diff
@@ -23,9 +24,9 @@ object HashJvmSpec extends DefaultRunnableSpec {
   def spec: ZSpec[Environment, Failure] =
     suite("HashJvmSpec")(
       suite("laws")(
-        testM("parMap")(checkAllLaws(Hash)(Gen.mapOf(Gen.anyInt, Gen.anyInt).map(_.par))),
-        testM("parSeq")(checkAllLaws(Hash)(Gen.listOf(Gen.anyInt).map(_.par))),
-        testM("parSet")(checkAllLaws(Hash)(Gen.setOf(Gen.anyInt).map(_.par)))
+        testM("parMap")(checkAllLaws(HashLaws)(Gen.mapOf(Gen.anyInt, Gen.anyInt).map(_.par))),
```

Comment:
Why `HashLaws` and not `laws.Hash`?

Reply:
I think that creates an undesirable name clash between the functional abstraction `Hash` and the laws `Hash`.

Reply:
Oh, I guess I'm that kind of a person that would be OK with having in `zio.prelude` `Hash` and `laws.Hash` -- having `laws.HashLaws` seems a bit redundant -- but it's OK too :wink:

Reply:
Definitely a matter of personal style but if you do `import zio.prelude._` and `import zio.prelude.laws._` you are going to have problems so I think best to avoid that.